src/components/Components/LinkSecret/SecretSelectMenu.tsx

```diff
@@ -0,0 +1,187 @@
+import React, { SetStateAction, useEffect } from 'react';
+import {
+  Divider,
+  MenuGroup,
+  MenuItem,
+  MenuList,
```

Comment:
The file has more than 90% same content with src/shared/components/component-select-menu/ComponentSelectMenu.tsx, is it possible to reuse ComponentSelectMenu?

Reply:
How about improve ComponentSelectMenu like: adding one optional parameter: linkedSecrets to the original parameter list.